com.bambulab.BambuStudio.yml

```diff
@@ -0,0 +1,382 @@
+app-id: com.bambulab.BambuStudio
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+command: entrypoint
+separate-locales: true
+finish-args:
+  - --share=ipc
+  #- --socket=wayland
+  #- --socket=fallback-x11
+  - --socket=x11
+  - --share=network
+  - --device=all
+  - --filesystem=home
+  - --filesystem=xdg-run/gvfs
+  - --filesystem=/run/media
+  - --filesystem=/media
+  # Allow BambuStudio to talk to other instances
+  - --talk-name=com.bambulab.BambuStudio.InstanceCheck.*
```

Comment:
I do believe `com.bambulab.BambuStudio.*` is granted based on the app-id.

Reply:
If it does, we need something equivalent to the `finish-args-unnecessary-appid-own-name` linter error to detect its usage.

Reply:
Can you please open an issue for this? This is not very clear in https://docs.flatpak.org/en/latest/flatpak-command-reference.html# > Session Bus Policy also but pretty sure owning one's own bus implies talking with itself.

Reply:
from the source code:
https://github.com/flatpak/flatpak/blob/main/common/flatpak-context.c#L2364

Reply:
> I do believe com.bambulab.BambuStudio.* is granted based on the app-id.

I read through `dbus-proxy/flatpak-proxy.c` and it grants `talk` based on the service's own name, not based on the app-id. So my `com.bambulab.BambuStudio.InstanceCheck.Object123` service can only talk to `com.bambulab.BambuStudio.InstanceCheck.Object123` and not to `com.bambulab.BambuStudio.InstanceCheck.Object789`.

```C
[...] The default
 * initial policy is that the the user is only allowed to TALK to the
 * bus itself (org.freedesktop.DBus, or no destination specified), and
 * TALK to its own unique id.
```

```C
             case EXPECTED_REPLY_HELLO:
              /* When we get the initial reply to Hello, allow all
                 further communications to our own unique id. */
              if (header->type == G_DBUS_MESSAGE_TYPE_METHOD_RETURN)
                {    
                  g_autofree char *my_id = get_arg0_string (buffer);
                  flatpak_proxy_client_update_unique_id_policy (client, my_id, FLATPAK_POLICY_TALK);
                  break;
                }    
```

So I guess that the code in the manifest is correct, and that I can resolve the conversation without opening an issue against the linter.